components/landing/TheQuickStart/PrerequisitesForLinux.vue

```diff
@@ -1,28 +1,15 @@
 <template>
   <div class="prerequisites">
     <div class="prerequisites__section">
-      <h3 class="h4">
-        Install the compiler
-      </h3>
-      <p>
-        If you’re using Fedora or an equivalent Linux distribution, install
-        using:
-      </p>
-      <pre class="code prerequisites__code">dnf install @development-tools</pre>
-      <p>For Ubuntu/Debian install it using:</p>
-      <pre class="code prerequisites__code">apt-get install build-essential</pre>
-    </div>
-    <div class="prerequisites__section">
-      <h3 class="h4">
-        Install OpenBLAS development headers
-      </h3>
+      <p>Run the following command and follow the onscreen instructions:</p>
```

Comment:
Since the prerequisites for Mac and Linux are the same, you can reuse the component

Reply:
Done here https://github.com/Qiskit/qiskit.org/pull/2904/commits/e4f624245eefd7dc5dc2fbe21522256669d6e091